frontend/src/components/InstanceForm.vue

```diff
@@ -292,16 +292,39 @@
           </div>
         </template>
 
-        <template v-if="state.instance.engine == 'MONGODB'">
-          <div class="mt-2 sm:col-span-1 sm:col-start-1">
-            <div class="lex flex-row items-center space-x-2">
-              <BBCheckbox
-                title="DNS SRV Record"
-                :value="currentDataSource.options.srv"
-                @toggle="handleToggleSRV"
-              />
-            </div></div
-        ></template>
+        <div
+          v-if="state.instance.engine === 'MONGODB'"
+          class="sm:col-span-4 sm:col-start-1"
+        >
+          <label
+            for="connectionStringSchema"
+            class="textlabel flex flex-row items-center"
+          >
+            {{ $t("data-source.connection-string-schema") }}
+          </label>
+          <label
+            v-for="type in mongodbConnectionStringSchemata"
+            :key="type"
+            class="radio h-7"
+          >
+            <input
+              type="radio"
+              class="btn"
+              name="connectionStringSchema"
+              :value="type"
+              :checked="
+                type ===
+                (currentDataSource.options.srv === false
+                  ? mongodbConnectionStringSchemata[0]
+                  : mongodbConnectionStringSchemata[1])
```

Comment:
This might be a little bit too fancy. How about extracting this as a `computed`?

Reply:
Done.